src/model.js

```diff
@@ -724,6 +724,17 @@ export class TokenID extends FixedSizedB58Str {
     return this.data === this.constructor.TESTNET_VSYS_TOK_ID;
   }
 
+  /**
+   * tokIdx returns the token index from the token id.
+   * @returns {TokenIdx} The token index.
+   */
+  get tokIdx() {
+    const tokIdBytes = bs58.decode(this.data);
```

Comment:
As `TokenID` is eventually a subclass of `B58Str`, it has already the attribute `bytes`. The code can be simplified as below.
```javascript
const tokIdNoChecksum = this.bytes.slice(0, -4);
const tokIdxBytes = tokIdNoChecksum.slice(-4);
return new TokenIdx(bp.unpackUInt32(tokIdxBytes));
```